weave

```diff
@@ -949,6 +949,12 @@ proxy_args() {
     done
 }
 
+proxy_autoconfig() {
+    if command_exists proxyvolumes ; then
+        PROXY_VOLUMES=${PROXY_VOLUMES:-$(proxyvolumes "$@")}
+    fi
+}
```

Comment:
I *think* this should work:

`! command_exists proxyvolumes || PROXY_VOLUMES=${PROXY_VOLUMES:-$(proxyvolumes "$@")}`